### PR TITLE
fix(FOROME-904): redesign main-table control panel

### PR DESCRIPTION
**Pull request by Lapk1n**
https://quantori.atlassian.net/browse/FOROME-904

In addition:
1. Sort ws-page folders structure
2. Refactoring for zone items

### CODE REVIEW

**Review by QYaroslavTrefilov — APPROVED** [review]
These 3 buttons do nothing. Maybe we should hide them for now?
image

**Comment by Lapk1n** [comment]
Yep, that's right

File: src/pages/filter/ui/empty-selected-group.tsx [hunk 1]
```diff
@@ -1,20 +1,28 @@
-import { ReactElement } from 'react'
+import { ReactElement, useRef } from 'react'
 import cn, { Argument } from 'classnames'
 
+import useClientHeight from '@core/hooks/use-client-height'
 import { t } from '@i18n'
 
 type Props = {
```
Comment (AndreyUstyumenko) [inline src/pages/filter/ui/empty-selected-group.tsx]: IEmptySelectedGroupProps

File: src/pages/ws/ui/control-panel/customize-table-modal/customize-table-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        onClearAll={() => columnsStore.clearAllColumns()}
+        onApply={() => {
+          columnsStore.filterColumns()
+          columnsStore.setViewType(viewType)
+          close()
+        }}
+        onClose={() => {
+          columnsStore.resetSearchColumnValue()
+          columnsStore.resetColumns()
+          close()
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/control-panel/customize-table-modal/customize-table-modal.tsx]: better to move to handler funcs

File: src/pages/ws/ui/control-panel/zone-modals/genes-list-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        datasetStore.fetchZoneListAsync('Panels')
+
+      if (zoneStore.selectedGenesList.length > 0) {
+        zoneStore.syncSelectedAndLocalFilters('isGenesList')
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [])
+
+    const handleApplyAsync = async () => {
+      close()
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/control-panel/zone-modals/genes-list-modal.tsx]: move close down to the bottom, we close modals after everything's gone well

File: src/pages/ws/ui/control-panel/zone-modals/genes-modal.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    datasetStore.genes.length <= 0 && datasetStore.fetchZoneListAsync('Symbol')
+
+    if (zoneStore.selectedGenes.length > 0) {
+      zoneStore.syncSelectedAndLocalFilters('isGenes')
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  const handleApplyAsync = async () => {
+    close()
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/control-panel/zone-modals/genes-modal.tsx]: same

File: src/pages/ws/ui/control-panel/zone-modals/samples-modal.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    datasetStore.samples.length <= 0 && datasetStore.fetchSamplesZoneAsync()
+
+    if (zoneStore.selectedSamples.length > 0) {
+      zoneStore.syncSelectedAndLocalFilters('isSamples')
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  const handleApplyAsync = async () => {
+    close()
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/control-panel/zone-modals/samples-modal.tsx]: same

File: src/pages/ws/ui/control-panel/zone-modals/tags-modal.tsx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+    if (zoneStore.selectedTags.length > 0) {
+      zoneStore.syncSelectedAndLocalFilters('isTags')
+    }
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  const handleApplyAsync = async () => {
+    close()
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/control-panel/zone-modals/tags-modal.tsx]: same

**Review by AndreyUstyumenko — APPROVED** [review]

File: src/pages/ws/ui/control-panel/zone-modals/components/header-table-button.tsx [hunk 7]
```diff
@@ -1,36 +1,30 @@
+import { ReactElement } from 'react'
 import cn, { Argument } from 'classnames'
 
-import { Icon } from '@ui/icon'
-
 interface Props {
```
Comment (AlMaQntr) [inline src/pages/ws/ui/control-panel/zone-modals/components/header-table-button.tsx]: IHeaderTableButtonProps

**Review by AlMaQntr — APPROVED** [review]

**Pull request merged**